src/main/java/rs/hybridit/model/User.java

```diff
@@ -39,12 +51,18 @@
 
 	private String username;
 
+	@JsonIgnore
 	private String password;
 
 	@JsonIgnore
 	@OneToMany(mappedBy = "user", fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
 	private Set<BookCopy> bookCopies = new HashSet<BookCopy>();
 
+	@ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
if you remove a user are you sure that you want to remove all related authorities?

Reply:
Changed it to PERSIST.